db/migrate/20160801053501_add_confirmed_at_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddConfirmedAtToUsers < ActiveRecord::Migration[5.0]
```

Comment:
@maxious I can fix it in the DB but it does mean that we will *never* send confirmation emails for existing users. Are we OK with that?